shader-minifier-linux.fsproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We can't use the same fsproj as on Windows?

Reply:
The current Windows fsproj is way too old and uses dotnet 4.8.
But maybe if we try to use the linux fsproj on Windows, it might work as is with dotnet 8?